src/dbos-executor.ts

```diff
@@ -1414,6 +1414,10 @@ export class DBOSExecutor implements DBOSExecutorContext {
       return check as R;
     }
 
+    if (this.debugMode) {
+      throw new DBOSDebuggerError(`Failed to debug workflow UUID: ${wfCtx.workflowUUID}`);
```

Comment:
Same as above, the error message could be clearer: "Couldn't find recorded output" or so.

Reply:
I will update the error messages.